.github/workflows/ci.yaml

```diff
@@ -0,0 +1,39 @@
+name: ci
+on:
+  pull_request:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        postgres:
+        - version: "14"
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install latest nightly
+        uses: actions-rs/toolchain@v1
+        with:
+            toolchain: stable
+            override: true
+
+      - uses: Swatinem/rust-cache@v1
+
+      - name: Install cargo-pgx
+        uses: actions-rs/cargo@v1
+        with:
+          command: install
+          args: cargo-pgx
+
+      - name: Initialize pgx
```

Comment:
It would be great if we could cache this step somehow. Also for the linting workflow.

Reply:
Good call, I think we should be able to cache the `~/.pgx` directory and be more or less good to go, I'll give that a shot here in a few